discord-haskell.cabal

```diff
@@ -88,7 +151,9 @@ library
     , Discord.Internal.Types.Emoji
     , Discord.Internal.Types.ScheduledEvents
   build-depends:
-    base ==4.*,
+  -- https://gitlab.haskell.org/ghc/ghc/-/wikis/commentary/libraries/version-history
+  -- below also sets the GHC version effectively. set to == 8.10.* alone for now.
+    base == 4.14.*,
```

Comment:
I'm not sure if this is the best idea because our current code builds and runs fine on 9.0.2 as long as base64-bytestring's upper bound is raised.

Maybe that's a better solution? (Not sure about 9.2.4, since there was no Stackage LTS for it, and I didn't want to mess around with my env too much)

Reply:
I can relax the bounds by adding `|| == 4.whatever.*` for the 9.0.2 version.

Reply:
Fair enuff. Ok!